src/elements/content-uploader/ContentUploader.js

```diff
@@ -936,6 +938,15 @@ class ContentUploader extends Component<Props, State> {
         let view = '';
         if ((items && items.length === 0) || allItemsArePending) {
             view = VIEW_UPLOAD_EMPTY;
+        } else if (isPartialUploadEnabled && areAllItemsFinished) {
+            const filesToBeUploaded = items.filter(item => item.status === STATUS_COMPLETE);
```

Comment:
nit: this should be called `completedUploads`

Reply:
> nit: this should be called `completedUploads`

You mean the prop that we need to pass in -order for calling partial success of files uploaded as success?

Reply:
@kkrivanka-box Its( areAllItemsFinished) an existing variable which implies -> ```const areAllItemsFinished = items.every(
            uploadItem => uploadItem.status === STATUS_COMPLETE || uploadItem.status === STATUS_ERROR,
        );```
        which is what we want, to check if all the files have either been uploaded or throw error, and nothing is in pending stage.

Reply:
@kkrivanka-box Its( areAllItemsFinished) an existing variable which implies -> ```const areAllItemsFinished = items.every(
            uploadItem => uploadItem.status === STATUS_COMPLETE || uploadItem.status === STATUS_ERROR,
        );```
        which is what we want, to check if all the files have either been uploaded or throw error, and nothing is in pending stage.